wicket/tests/output/example_non_empty.toml

```diff
@@ -50,9 +50,7 @@ bootstrap_sleds = [
     5, # serial 4 5 6 (model model2 revision 5, ::1)
 ]
 
-# Allowlist of source IPs that can make requests to user-facing services.
 [allowed_source_ips]
-# Any external IPs to make requests. This is the default.
```

Comment:
Any particular reason the comments were removed?

Reply:
Also I think this is separate from this PR, but I really would like the format to be either `"any"` or a list of IPs.

Reply:
The comments were removed to placate the tests. I wasn't clear on how one could emit comments in the `TomlTemplate` alongside the allow-list itself. It could be that just moving them all prior to the table would solve it, I'm not sure.

> I really would like the format to be either "any" or a list of IPs/subnets.

Do you mean that the TOML itself would look like this?

```toml
allow = "any"
# OR
# allow = ["1.2.3.4/5"]
```

If so, then there's a tension here with the JSON schema for these API types. While it is the most natural for Rust, the generated schema is quite complicated and a bit awkward, since we effectively get an untagged enum of completely different types. I feel like this is a reasonable middle ground, but we can definitely bike-shed it a bit when it becomes important.

Reply:
> Any particular reason the comments were removed?

I've moved the comments all prior to the actual table itself, and now they all round-trip just fine. I'll put them back like that for documentation.

Reply:
Added the comments to the test in 0be6930b6

Reply:
> If so, then there's a tension here with the JSON schema for these API types.

Agreed. I think we should use separate deserializers for toml and JSON.